app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -50,12 +50,6 @@ class BrowserRobot {
     }
 
     fun verifyUrl(url: String) {
-        val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
-        mDevice.waitNotNull(
-            Until.findObject(By.res("org.mozilla.fenix.debug:id/mozac_browser_toolbar_url_view")),
```

Comment:
We will need to modify all these calls so that tests also work on beta. This only works on debug. But if we use just the id like `onView(withId(R.id.mozac_browser_toolbar_url_view))` it will work on beta and nightly

Reply:
@isabelrios how will we handle this so that we don't need 2 versions of each test?  If I recall correctly didn't @AaronMT or @sv-ohorvath say there was a way to parametrize the package name (if it's needed)?

Reply:
nvrmd, i see that you're one step ahead of me :)

Reply:
yeah, that was already implemented and working nicely :) with that we will be able to run tests using any package, debug, beta or nightly